rustls/src/conn.rs

```diff
@@ -271,6 +271,29 @@ mod connection {
         }
     }
 
+    impl BufRead for Reader<'_> {
+        /// Obtain a chunk of plaintext data received from the peer over this TLS connection.
+        /// This reads the same data as [`Reader::read`], but returns a reference instead of
```

Comment:
Nit: please refer to functions and methods with trailing `()` in order to distinguish from fields.